src/state/queries/profile.ts

```diff
@@ -253,6 +254,20 @@ export function useProfileFollowMutationQueue(
       updateProfileShadow(queryClient, did, {
         followingUri: finalFollowingUri,
       })
+
+      if (finalFollowingUri) {
+        agent.app.bsky.graph
+          .getSuggestedFollowsByActor({
+            actor: did,
+          })
+          .then(res => {
+            const dids = res.data.suggestions
+              .filter(a => !a.viewer?.following)
+              .map(a => a.did)
+              .slice(0, 8)
+            userActionHistory.followSuggestion(dids)
+          })
```

Comment:
Should we add a catch here and ignore the error?

Reply:
I think it'll be fine either way since it's async — we'll see an unhandled rejection at most (which should be fine on both web and RN). Is that correct?

Reply:
I'd prefer not to ignore in case we introduce a coding mistake somewhere. E.g. if there's a crash, I'd rather it get surfaced. And if we add a catch() that just calls the logger, that seems equivalent to not adding it at all. Lmk if this is wrong though

Reply:
Note that since it's not awaited, a network error wouldn't propagate to the calling code